src/layouts/ImageCard.css

```diff
@@ -10,13 +10,21 @@
     margin-right: 10px;
     background-color: floralwhite;
     color: darkslateblue;
-    text-overflow: ellipsis;
-    overflow: hidden;
     display: inline-block;
     white-space: pre-line;
 }
 
+.ImageCard > p {
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
I tried adding this cause google but i'm not sure if this is in the wrong place or i need a different property cause it's not really showing ellipsis even if i modify things :/

Reply:
Try removing white-space: pre-line from .ImageCard, adding it to .ImageCard > p as white-space: nowrap. Then you should see a single line with ellipses

Reply:
ooh it worked! i thought i tried some variation of this yesterday but maybe the sizing of something messed it up 🤔 
thanks!

Reply:
Ayy